changelogs/fragments/ec2_vpc_peering_facts_tags.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Changed output of tags dictionary in results by using boto3_tag_list_to_ansible_dict function
```

Comment:
I would just say something like "tags results have been changed to standard Ansible format"